### PR TITLE
Pinch and Zoom handling through atomic actions

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
Handling of pinch and zoom using atomic actions

@imurchie please review and advice if any.

@TikhomirovSergey FYI

### CODE REVIEW

**Comment by imurchie** [comment]
This looks reasonable to me. Travis is down for Mac builds right now. Once I'm done with a few things I'll try it out locally.

File: lib/commands/gesture.js [hunk 1]
```diff
@@ -124,6 +124,15 @@ helpers.isDrag = function (gestures) {
   );
 };
 
+helpers.isPinchAndZoom= function (gestures) {
```
Comment (jlipps) [inline lib/commands/gesture.js]: odd spacing here

File: lib/commands/gesture.js [hunk 2]
```diff
@@ -124,6 +124,15 @@ helpers.isDrag = function (gestures) {
...
+  return (
+      gestures.length === 3 &&
```
Comment (jlipps) [inline lib/commands/gesture.js]: 2-space indentation would be preferable

**Comment by jlipps** [comment]
couple style comments, otherwise LGTM too

**Comment by imurchie** [comment]
If you update the npm modules (`rm -rf node_modules && npm install`) the pre-commit hooks should run eslint, which will catch these.

**Comment by SrinivasanTarget** [comment]
@imurchie No issues found on running above command https://gist.github.com/SrinivasanTarget/335026f2939fab29e022888de46ed521

**Comment by imurchie** [comment]
But next time you commit it will run eslint and the unit tests beforehand.

You can manually run eslint with `gulp eslint`.

**Comment by SrinivasanTarget** [comment]
@imurchie Yup it ran actually

**Comment by SrinivasanTarget** [comment]
@imurchie @jlipps Travis is Green now. FYI I had an alternative solution using Native UIAutomation Methods using ```pinchOpenFromToForDuration ``` and ```pinchCloseFromToForDuration ``` which i tested across several real devices are inconsistent at times but atomic actions seems to be working fine on my several real devices.

**Pull request merged**